jwst/transforms/models.py

```diff
@@ -1450,21 +1447,23 @@ def evaluate(self, x, y, wavelength, order):
 
 
 class NIRISSForwardRowGrismDispersion(Model):
-    """This model calculates the dispersion extent of NIRISS pixels.
+    """This model calculates the wavelengths of vertically dispersed NIRISS grism data.
 
     The dispersion polynomial is relative to the input x,y pixels
     in the direct image for a given wavelength.
 
     Parameters
     ----------
+    orders : list
+        The list of orders which are available to the model
     xmodels : list[tuples]
         The list of tuple(models) for the polynomial model in x
     ymodels : list[tuples]
         The list of tuple(models) for the polynomial model in y
     lmodels : list
         The list of models for the polynomial model in l
-    orders : list
-        The list of orders which are available to the model
+    theta : float
```

Comment:
It looks like the ASDF Type to serialize this properly already has `theta`?  Was this just a case of Type being more up-to-date than the model itself?

`order` is removed because it is only used in the call to `evaluate`, whereas `theta` is needed on class init?

Reply:
This line is just adding `theta` to the docstrings of the model. it was used and defined but not in the docstrings. The real issue was with the unit tests only. They did not use `theta` and because the angle of rotation is defined as the difference between theta_ref and theta, when theta is 0 the trace in the unit tests was rotated by 33 deg.